api/spec/controllers/spree/api/line_items_controller_spec.rb

```diff
@@ -94,7 +94,9 @@ module Spree
         api_put :update, :id => line_item.id, :line_item => { :quantity => 101 }
         expect(response.status).to eq(200)
         order.reload
-        expect(order.total).to eq(1010) # 10 original due to factory, + 1000 in this test
+        subtotal = line_item.price * 101
+        shipping = order.shipments.first.cost
+        expect(order.total).to eq(subtotal + shipping)
```

Comment:
I debated that point. As a newcomer to the code I first found the numerical constant mysterious and had to break down in my head where it came from. My other option for this spec is to ensure the order itself is created without a selected shipping method, then I won't need to change the body of this test. I'll work on that as a compromise.

(I could just replace `1010` with `1110` to get it to pass, but that feels like cheating)